views/js/admin/vue.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+
+module.exports = {
+  chainWebpack: (config) => {
+    config.optimization.minimize(false)
+    config.plugins.delete('html');
+    config.plugins.delete('preload');
+    config.plugins.delete('prefetch');
+  },
+  configureWebpack: {
+    devtool: 'sourcemap',
+    output: {filename: '[name].js', chunkFilename: 'chunks/[name].js'},
+    resolve: {
+      alias: {
+        '@': path.resolve(__dirname, 'src'),
+      },
+    },
+  },
+  css: {
+    extract: false
+  },
+  runtimeCompiler: false,
+  productionSourceMap: false,
+  filenameHashing: false,
+  // Output: /<module>/views/dist/js/admin
+  outputDir: path.resolve(__dirname, '..', '..', 'dist', 'js', 'admin'),
+  assetsDir: '',
+  publicPath: '/modules/myparcelbe/views/dist/js/admin',
```

Comment:
zou je deze ook niet opbouwen als de outputdir?

Reply:
Nee dit is weer wat anders, dit bepaalt de url waarop de app (denkt dat 'ie) draait, is nodig voor file resolving